src/app/utils/dateUtils.ts

```diff
@@ -42,3 +42,17 @@ export const erUkeFørSammeEllerEtterDenneUken = (week: DateRange): 'før' | 'sa
     }
     return undefined;
 };
+
+export const søkerKunHelgedager = (fom?: string, tom?: string): boolean => {
+    if (fom && tom) {
+        const fomDayJs = dayjs(fom);
```

Comment:
DatePickerUtils har utils for å hente ut dato fra tekst som kommer fra input (getDateFromDateString) som du kanskje kan bruke